Shared/src/com/pacoapp/paco/shared/model2/OutputBaseColumns.java

```diff
@@ -0,0 +1,29 @@
+/*
+* Copyright 2011 Google Inc. All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance  with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+/**
+ *
+ */
+package com.pacoapp.paco.shared.model2;
+
+
+public class OutputBaseColumns {
+  public static final String EVENT_ID = "event_id";
+  public static final String INPUT_SERVER_ID = "input_server_id";
+  public static final String NAME = "text";
+  public static final String ANSWER = "answer";
+  public static final String ARCHIVE_FLAG = "archive_flag";
```

Comment:
When we decide to use partitions, the archive flag in outputs table will be more useful.